sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -188,7 +198,16 @@ def _infer_output_coder(self, input_type=None, input_coder=None):
   def expand(self, pbegin):
     assert isinstance(pbegin, pvalue.PBegin)
     self.pipeline = pbegin.pipeline
-    return pvalue.PCollection(self.pipeline, is_bounded=False)
+    if len(self.output_tags) == 0:
+      self.output_tags = set([None])
+
+    if len(self.output_tags) == 1:
```

Comment:
Is there a reason why we treat the case with only one output_tag as a special case (i.e. not returning a dict) ? Please add the reason as a comment in this code.

Reply:
For backwards compatibility, existing code already relies on a single PCollection being returned.